### PR TITLE
Environment beforeyouread

### PR DESCRIPTION
**Pull request by Xintong-Li**
Added a `beforeyouread` environment and a `quickcheck` command.

**Example. **
image
```
\begin{beforeyouread}
    \item Distinguishing between ``some'' and ``all''.
    \quickcheck{Write down a English word so that some of letters are vowels. Write down another English word so that all letters 
            are vowels.}
    \item Visualizing vector addition and scalar multiplication.
    \quickcheck{Draw the vector $\vec v_{1} = (1,0)$, $\vec v_{2} = (1,1)$, $\vec v_1+\vec v_2$, and the set of all scalar multiples of
            $\vec v_{1}$.}
    \item Visualizing vector addition and scalar multiplication.
    \quickcheck{Draw the vector $\vec v_{1} = (1,0)$, $\vec v_{2} = (1,1)$, $\vec v_1+\vec v_2$, and the set of all scalar multiples of
            $\vec v_{1}$.}
\end{beforeyouread}
```

### CODE REVIEW

File: book/common/preamble.tex [hunk 1]
```diff
@@ -186,7 +186,15 @@
 	}\xspace%
 }
 
-
+\newenvironment{beforeyouread}{
```
Comment (siefkenj) [inline book/common/preamble.tex:189]: Use `DeclareDocumentEnvironment`

File: book/common/preamble.tex [hunk 2]
```diff
@@ -186,7 +186,15 @@
...
+Before you read, make sure you are comfortable with the following tasks.
+Please do the ``Quick Check'' problem to see if you are comfortable with each
+ask.
+\begin{itemize}
+}{
+\end{itemize}
+}
+\newcommand\quickcheck[1]{\vspace{-0.5cm}\item[] {\footnotesize \textsc{Quick Check.} \textrm{#1}}}
```
Comment (siefkenj) [inline book/common/preamble.tex:197]: Don't use an `\item` for this. Just put in a `\par` at the start. (Using an `\item` will mess things up when we change the item spacing)

**Comment by Xintong-Li** [comment]
Update.

Now it looks like this.
image

File: book/common/preamble.tex [hunk 3]
```diff
@@ -186,7 +186,15 @@
 	}\xspace%
 }
 
-
+\DeclareDocumentEnvironment{beforeyouread}{}{
+Before you read, make sure you are comfortable with the following tasks.
+Please do the ``Quick Check'' problem to see if you are comfortable with each
+ask.
```
Comment (siefkenj) [inline book/common/preamble.tex:192]: ask -> task

File: book/common/preamble.tex [hunk 4]
```diff
@@ -186,7 +186,15 @@
 	}\xspace%
 }
 
-
+\DeclareDocumentEnvironment{beforeyouread}{}{
+Before you read, make sure you are comfortable with the following tasks.
```
Comment (siefkenj) [inline book/common/preamble.tex:190]: -> `comfortable with the following.` Remove the word "task"

**Pull request merged**